R/mread.R

```diff
@@ -423,11 +430,14 @@ mread <- function(model, project = getOption("mrgsolve.project", getwd()),
   check_globals(mread.env[["move_global"]], Cmt(x))
   
   # This must come after audit
+  # TODO: harmonize with other audit process
   if(!has_name("ODE", spec)) {
     spec[["ODE"]] <- "DXDTZERO();"
-  } else if(mread.env[["audit_dadt"]]) {
-    spec[["ODE"]] <- c(spec[["ODE"]], ode)
-    audit_spec(x, spec, warn = warn)
+  } else {
+    if(mread.env[["audit_dadt"]] && !mread.env[["using_nm-vars"]]) {
```

Comment:
This fixes the audit issue that we found later on; I have on TODO list to harmonize with other audits.